cmd/proxy/actions/app_proxy_test.go

```diff
@@ -0,0 +1,94 @@
+package actions
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/gomods/athens/pkg/build"
+	"github.com/gomods/athens/pkg/config"
+	"github.com/gomods/athens/pkg/log"
+	"github.com/gomods/athens/pkg/storage/mem"
+	"github.com/gorilla/mux"
+	testlog "github.com/sirupsen/logrus/hooks/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type routeTest struct {
+	method string
+	path   string
+	body   string
+	test   func(t *testing.T, resp *http.Response)
+}
+
+func TestProxyRoutes(t *testing.T) {
+	r := mux.NewRouter()
+	s, err := mem.NewStorage()
+	require.NoError(t, err)
+	lggr, hook := testlog.NewNullLogger()
+	require.NotNil(t, hook)
+	l := &log.Logger{Logger: lggr}
+	c, err := config.Load("")
+	require.NoError(t, err)
+	c.NoSumPatterns = []string{"*"} // catch all patterns with noSumWrapper
+	c.PathPrefix = "/prefix"
+	subRouter := r.PathPrefix(c.PathPrefix).Subrouter()
+	err = addProxyRoutes(subRouter, s, l, c)
+	require.NoError(t, err)
+
+	baseURL := "https://athens.azurefd.net/prefix"
```

Comment:
```suggestion
	baseURL := "https://athens.azurefd.net" + c.PathPrefix
```

This way, someone else reading the code will recognize that `/prefix` is an intentional part of the test, and that if we needed to change it, we'll just have to change it once on line 38